### ISSUE TITLE
Issue with sequelize and instana

### ISSUE THREAD

**Issue by akoufatzis** [instana/nodejs-sensor#125]
Hello, I use Postgres and instana. I have the following issue: [https://github.com/sequelize/sequelize/issues/10521](https://github.com/sequelize/sequelize/issues/10521) but I found out that it is related to instana because if I disable tracing everything works fine. I also observed that I hit the connection limit and it seems that every sequelize query that is instrumented by instana opens a new connection.

I am using the latest version of the nodejs-sensor package: "v1.61.2" .
Is it possible to just disable tracing for **pg** ?

Thank you in advance

**Comment by basti1302** [comment]
Hi,

no, it is not possible to only disable only a single intrumentation. I'll look into this.

Thanks for the report.

**Comment by akoufatzis** [comment]
Thanks for looking at it. I could add the ability to disable only a single instrumentation if you want and open a PR.

**Comment by basti1302** [comment]
Hi,

I'm not entirely sure if I this _should_ be a feature that this package offers. Under normal circumstances, it should just not be needed. Although I agree that it would come in very handy in situations like this, where one particular instrumentations causes trouble.

How would you design the API to disable particular instrumentations?

**Comment by akoufatzis** [comment]
Hi,

as I see at the moment I only can use `config.tracing.tracing` and/or `config.tracing.disableAutomaticTracing` to disable tracing and instrumentation for frameworks, databases etc under the `instrumentation` folder. This would be the default way, but there could be an option to pass an options object too, that has keys like: postgres, express etc. This would overwrite the previous global flags.

**Comment by basti1302** [comment]
That sounds about right. 

How about:

```javascript
require('instana-nodejs-sensor')({
  tracing: {
    blacklist: ['pg', 'mongodb']
  }
});
```

So instead of an _object_ with keys I would suggest using an _array of strings_ that lists the instrumentations that should be disabled. Not sure if `blacklist` is really the best name for it.

The keys in that list probably should be the module names in `src/tracing/instrumentation`.

In other news, I think I found the issue that you encountered with `sequelize`, I'll probably release a fix for that later today or early tomorrow.

**Comment by basti1302** [comment]
The fix has landed in `1.63.0`.

**Comment by akoufatzis** [comment]
Yes instead of blacklist maybe the name `skipTracing` would be cleaner for this use case. Thank you for the new version. I will use it and check if the issue persists or not.
